include/geometry/open_lines_set.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2024 UltiMaker
+// CuraEngine is released under the terms of the AGPLv3 or higher
+
+#ifndef GEOMETRY_OPEN_LINES_SET_H
+#define GEOMETRY_OPEN_LINES_SET_H
+
+#include "geometry/lines_set.h"
+#include "geometry/open_polyline.h"
+
+namespace cura
+{
+
+/*!
+ * \brief Convenience definition for a container that can hold only open polylines. This makes it
+ *        explicit what the lines actually represent.
+ */
+using OpenLinesSet = LinesSet<OpenPolyline>;
```

Comment:
Indeed. I wanted to do that, but I use forward declarations as much as possible to avoid including too many headers into headers. And you can't forward-declare a `using` :cry: so in headers I have to use `LinesSet<OpenPolyline>` then in the cpp file I can include the proper header and use this name. I couldn't find a better way without including the files.
Or maybe I can put a forward declaration in this header, I will try that.

Reply:
That strategy :point_up:  did work, and even allowed for less header inclusion into headers. :+1: